### PR TITLE
[Snyk] Upgrade tsx from 3.4.0 to 3.4.2

### PR DESCRIPTION
**Pull request by alexec**
<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to upgrade tsx from 3.4.0 to 3.4.2.</h3>

merge advice
:information_source: Keep your dependencies up-to-date. This makes it easier to fix existing vulnerabilities and to more quickly identify and fix newly disclosed vulnerabilities when they affect your project.
<hr/>

- The recommended version is **2 versions** ahead of your current version.
- The recommended version was released **22 days ago**, on 2022-05-25.

<hr/>

**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open upgrade PRs.*

For more information:  

🧐 [View latest project report](https://app.snyk.io/org/argoproj/project/d2931792-eef9-4d7c-b9d6-c0cbd2bd4dbe?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr)

🛠 [Adjust upgrade PR settings](https://app.snyk.io/org/argoproj/project/d2931792-eef9-4d7c-b9d6-c0cbd2bd4dbe/settings/integration?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr)

🔕 [Ignore this dependency or unsubscribe from future upgrade PRs](https://app.snyk.io/org/argoproj/project/d2931792-eef9-4d7c-b9d6-c0cbd2bd4dbe/settings/integration?pkg&#x3D;tsx&amp;utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr#auto-dep-upgrades)

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "rxjs": "^6.6.6",
     "superagent": "^7.1.3",
     "timezones-list": "3.0.1",
-    "tsx": "^3.4.0",
+    "tsx": "^3.4.2",
```
Comment (crenshaw-dev) [inline ui/package.json:48]: I think this should be moved to devDependencies.

Comment (alexec) [reply ui/package.json:48]: Let’s the bot know?

Comment (crenshaw-dev) [reply ui/package.json:48]: Sorry, not sure I understand the question.

Comment (crenshaw-dev) [reply ui/package.json:48]: Oh lol gotcha. Was just commenting for anyone tempted to test/merge this.  :-P

But agreed, Snyk should be funnier. 0/10 on comedy.

**Pull request closed without merging**